src/main/models/terminal/TerminalWorker.java

```diff
@@ -0,0 +1,41 @@
+package main.models.terminal;
+
+import main.models.terminal.interfaces.Display;
+import main.models.terminal.interfaces.Park;
+import main.models.vehicles.HybridCar;
+import main.models.vehicles.Vehicle;
+
+public class TerminalWorker implements Park, Display {
+    private final String name;
+    private double cardPercentage = 10.0;
+    private double salary = 0;
+
+    public TerminalWorker(final String name){
+        this.name = name;
+    }
+    public TerminalWorker(final String name, final double cardPercentage){
+        this.name = name;
+        this.cardPercentage = cardPercentage;
+    }
+
+    @Override
+    public boolean parkVehicle(Vehicle vehicle, Terminal terminal) {
```

Comment:
I see your point of segregating each functionality into interfaces but don't be afraid of using superclasses or abstract superclasses with methods that can or must be overiden!